ovn-tester/ovn_utils.py

```diff
@@ -101,6 +101,29 @@ def external_host_provision(self, ip, gw, netns='ext-ns'):
         self.run(cmd=cmd)
 
 
+class NodeConf:
```

Comment:
Isn't this exactly the same as just defining a named tuple?  E.g.:
```
NodeConf = namedtuple('NodeConf', ['mgmt_net', 'int_net', 'ext_net', 'gw_net'])
```

We won't have getters but that's fine I think.

Reply:
@dceara ack, will do